tests/x11regressions/firefox/firefox_java.pm

```diff
@@ -53,13 +53,21 @@ sub java_testing {
 
 sub run() {
     my ($self) = @_;
-    $self->start_firefox;
-
-    assert_and_click('firefox-logo');
+    # Need to change owner of serial devide on opensuse
+    $self->start_firefox(check_var('DISTRI', 'opensuse'));
+    #Required only on sle, as open mozilla home page
```

Comment:
Somehow start_firefox method for opensuse only starts the browser, and home page is not set to mozilla home page, but to search page. I'm not sure why this check is there for sle, as not related to java test at all.